sass/components/_button.scss

```diff
@@ -36,4 +32,43 @@
       outline: 3px solid $color-secondary-light-1;
     }
   }
+
+  &__light-green {
```

Comment:
Could you refactor classes _light-green and dark_green, I think is makes more sense to have something like:

```sass
&__green {
   &-light{  }
   &-dark{ }
 }
```

This way we will have less code repetition and code will be more easy to maintain.

Reply:
Good catch, you are right, is going to be better this way. Thank you